plugins/modules/aci_fabric_node.py

```diff
@@ -46,6 +46,18 @@
     type: str
     aliases: [ role_name ]
     choices: [ leaf, spine, unspecified ]
+  node_type:
```

Comment:
please add your contact details to top of the file and to the author section

Reply:
Do not replace but keep the original and add yourself on new line, and also add to author section ( line 68 )